File: app/src/main/java/com/example/spacetraders/data/views/TitleActivity.java
package com.example.spacetraders.data.views;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

import com.example.spacetraders.R;


public class TitleActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_title);

        /*
         * Disables actionbar back button
         */
        if (getSupportActionBar() != null) {
            getSupportActionBar().setDisplayHomeAsUpEnabled(false);
            getSupportActionBar().setHomeButtonEnabled(false);
        }

        Button button1 = findViewById(R.id.button_newGame);

        button1.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                Intent intent = new Intent(TitleActivity.this, NewCharacterActivity.class);
                startActivity(intent);
            }
        });

        Button button2 = findViewById(R.id.button_continueGame);

        button2.setOnClickListener(new View.OnClickListener() {
            public void onClick(View v) {
                Intent intent = new Intent(TitleActivity.this, OldCharacterActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: app/src/main/java/com/example/spacetraders/data/views/TitleActivity.kt
package com.example.spacetraders.data.views

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.view.View
import android.widget.Button

import com.example.spacetraders.R


class TitleActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_title)

        /*
         * Disables actionbar back button
         */
        if (supportActionBar != null) {
            supportActionBar!!.setDisplayHomeAsUpEnabled(false)
            supportActionBar!!.setHomeButtonEnabled(false)
        }

        val button1 = findViewById<Button>(R.id.button_newGame)

        button1.setOnClickListener {
            val intent = Intent(this@TitleActivity, NewCharacterActivity::class.java)
            startActivity(intent)
        }

        val button2 = findViewById<Button>(R.id.button_continueGame)

        button2.setOnClickListener {
            val intent = Intent(this@TitleActivity, OldCharacterActivity::class.java)
            startActivity(intent)
        }
    }
}